webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux.js';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo.json';
+
+// import from a file named `bar.js` located in the parent directory:
+import f3 from '../bar.js';
```

Comment:
close! check out [import practice](https://github.com/Learning-Fuze/curriculum/tree/faf2f0d2a513ce447cd0759441d90f967500b02a/exercises/webpack-and-babel#import-practice) important note